deployment/data-streams/changeset/sequence/deploy_data_streams.go

```diff
@@ -0,0 +1,312 @@
+package sequence
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/mcms"
+
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/utils/mcmsutil"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset"
+	feemanager "github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/fee-manager"
+	rewardmanager "github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/reward-manager"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/types"
+	"github.com/smartcontractkit/chainlink/deployment/data-streams/changeset/verification"
+	dsutil "github.com/smartcontractkit/chainlink/deployment/data-streams/utils"
+)
+
+// DeployDataStreamsDestinationChainChangeset deploys the entire data streams destination chain contracts. It should be kept up to date
+// with the latest contract versions and deployment logic.
+var DeployDataStreamsDestinationChainChangeset = deployment.CreateChangeSet(deployDataStreamsLogic, deployDataStreamsPrecondition)
+
+type DeployDataStreamsConfig struct {
+	ChainsToDeploy map[uint64]DeployDataStreams
+}
+
+type BillingConfig struct {
+	LinkTokenAddress   common.Address
+	NativeTokenAddress common.Address
+	Surcharge          uint64
```

Comment:
Will these values be documented separately in a runbook or in code? 

Might be useful to have comments explaining some less intuitive attributes directly on the struct

Reply:
We'll have default configs that will populate this